devdocs/process.md

```diff
@@ -70,7 +70,7 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
        git checkout master
        git pull upstream master       
        //merge updates into working branch
-       git checkout Issue{issue number}
+       git checkout {IssueNumber}-{some-keywords}
```

Comment:
In some places we can use `{branch-name}` instead of repeating the naming convention? Otherwise good to merge.